src/models.py

```diff
@@ -496,52 +527,60 @@ def build_reddit_module(task, d_inp, params):
 
 
 def build_image_sent_module(task, d_inp, params):
-    pooler = Pooler.from_params(d_inp, params['d_proj'])
+    pooler = Pooler(d_inp, d_proj=params['d_proj'])
     return pooler
 
 
-def build_single_sentence_module(task, d_inp, params):
+def build_single_sentence_module(task, d_inp, model, params):
```

Comment:
Pass the minimum information needed to reduce leaky abstractions? (instead of `model`, just something like `should_proj=not model.use_bert`)